README.rst

```diff
@@ -1,64 +1,241 @@
+          the `STUPS documentation`_. We have `tools`_ that can generate
+          this file for you.
+
+After the image build, it will be available within the tag
+"lizzy:dev". You will also need to specify some environment
+variables to be used by Lizzy. Here is an "example.cfg" file:
+
+.. code-block:: cfg
+
+    TOKEN_URL=https://token.auth.example.com/oauth2/access_token
+    TOKENINFO_URL=https://info.auth.example.com/oauth2/tokeninfo
+    ALLOWED_USERS=['robotusername','myusername']
+    DEPLOYER_SCOPE=deployer
+    LOG_LEVEL=DEBUG
+
+It is also necessary to configure the `AWS credentials`_ locally in
+your machine under `~/.aws`. After that you can run the Lizzy image
+with the command:
 
+.. code-block:: sh
+
+    $ docker run --rm -p 8080:8080 --env-file environment.cfg -v ~/.aws:/.aws --name lizzy -u 999 -it lizzy:dev
+
+The application by default will be listening on port `8080`. Usually
+accessible at `http://127.0.0.1:8080`, it depends on how your Docker is
+configured locally. A Swagger/OpenAPI console will be available at
+`http://127.0.0.1:8080/api/ui/#/default`.
+
+Deploying to AWS
+================
+
+There are many ways to deploy Lizzy to AWS. At Zalando we use `STUPS
+platform`_ that provides a convenient and audit-compliant
+Platform-as-a-Service (PaaS). An example of the `Senza definition`_ to
+deploy Lizzy would be:
+
+.. code-block:: yaml
+
+    SenzaInfo:
+      StackName: lizzy
+      Parameters:
+        - ImageVersion:
+            Description: "Docker image version of lizzy."
+    SenzaComponents:
+      - Configuration:
+          Type: 'Senza::StupsAutoConfiguration'
+      - AppServer:
+          Type: Senza::TaupageAutoScalingGroup
+          AssociatePublicIpAddress: false
+          ElasticLoadBalancer: AppLoadBalancer
+          IamRoles: ['app-lizzy']
+          InstanceType: t2.nano
+          SecurityGroups: ['app-lizzy']
+          TaupageConfig:
+            application_version: '{{Arguments.ImageVersion}}'
+            environment:
+              ALLOWED_USER_PATTERN: "^(jenkins-slave-\\w+)$"
+              DEPLOYER_SCOPE: myscope
+              LANG: C.UTF-8
+              LC_ALL: C.UTF-8
+              LOG_LEVEL: DEBUG
+              REGION: '{{AccountInfo.Region}}'
+              TOKEN_URL: 'https://token.auth.example.com/oauth2/access_token'
+              TOKENINFO_URL: 'https://info.auth.example.com/oauth2/tokeninfo'
+            health_check_path: /api/swagger.json
+            ports: {8080: 8080}
+            runtime: Docker
+            source: 'lizzy:{{Arguments.ImageVersion}}'
+      - AppLoadBalancer:
+          HTTPPort: 8080
+          HealthCheckPath: /api/swagger.json
+          Scheme: internet-facing
+          SecurityGroups: ['app-lizzy-lb']
+          Type: Senza::WeightedDnsElasticLoadBalancer
+
+Access Control for Lizzy
+------------------------
+
+Lizzy application will need to have access to create new Cloud
```

Comment:
I feel the sentence is slightly off and it should be
> Lizzy application should have access to create new...

the use two `to` in close proximity feels awkward. I might be mistaken here though.